test/test_image.py

```diff
@@ -210,56 +212,44 @@ def test_decode_image(self):
             self.assertTrue(img_lpng.equal(img_pil))
 
     def test_read_file(self):
-        with get_tmp_dir() as d:
```

Comment:
These tests that use the `get_tmp_dir()` method are actually fine. We should probably keep this logic and a use it in other places.